server/routes/incidentDetails/incidentDetailsEdit.ts

```diff
@@ -135,17 +142,26 @@ export default class IncidentDetailsEditRoutes {
     } = req.body
     const { user } = res.locals
     const { prisonerNumber, id } = req.params
+    const { personDeleted } = req.query
     const selectedPerson = JSON.stringify(req.query.selectedPerson)?.replace(/"/g, '')
     const { originalRadioSelection } = req.session
 
+    const newlySelectedAssociatedPrisonersNumber =
```

Comment:
Minor thing, and it might not be possible, but we should probably have the 1search` code above this as it does not need this info, does it?